cypress/e2e/gui/engageSphere.cy.js

```diff
@@ -0,0 +1,141 @@
+      .and(
+        'have.attr',
+        'href',
+        'https://open.spotify.com/show/5HFlqWkk6qtgJquUixyuKo'
+      )
+      .and('have.attr', 'target', '_blank')
+      .and('have.attr', 'rel', 'noopener noreferrer');
+
+    cy.contains('a', 'Courses')
+      .should('be.visible')
+      .and('have.attr', 'href', 'https://talking-about-testing.vercel.app/')
+      .and('have.attr', 'target', '_blank')
+      .and('have.attr', 'rel', 'noopener noreferrer');
+
+    cy.contains('a', 'Blog')
+      .should('be.visible')
+      .and('have.attr', 'href', 'https://talkingabouttesting.com')
+      .and('have.attr', 'target', '_blank')
+      .and('have.attr', 'rel', 'noopener noreferrer');
+
+    cy.contains('a', 'YouTube')
+      .should('be.visible')
+      .and('have.attr', 'href', 'https://youtube.com/@talkingabouttesting')
+      .and('have.attr', 'target', '_blank')
+      .and('have.attr', 'rel', 'noopener noreferrer');
+  });
+
+  it('Exibe a saudação "Hi, there" quando nenhum nome é fornecido', () => {
+    cy.get('[data-testid="name"]').should('have.value', '');
+
+    cy.contains('h2', 'Hi there').should('be.visible');
+  });
+
+  it('Exibe a saudação "Hi, Joe" quando o nome é fornecido', () => {
+    cy.get('[data-testid="name"]').type('Joe');
+
+    cy.contains('h2', 'Hi Joe').should('be.visible');
+  });
+
+  it('Exibe o cabeçalho com um título, alternador de tema e um campo de entrada de texto', () => {
+    cy.contains('h1', 'EngageSphere').should('be.visible');
+    cy.getByClassThatStartsWith('ThemeToggle_button').should('be.visible');
+    cy.get('[data-testid="name"]').should('be.visible');
+  });
+
+  it('Abre e fecha o messenger', () => {
+    cy.getByClassThatStartsWith('Messenger_openCloseButton').click();
+    cy.get('#messenger-name').should('be.visible');
+    cy.getByClassThatStartsWith('Messenger_openCloseButton').click();
+    cy.get('#messenger-name').should('not.exist');
+  });
+
+  it('Garante que todos os campos do messenger são obrigatórios e que o primeiro está focado', () => {
+    cy.getByClassThatStartsWith('Messenger_openCloseButton').click();
+    
+    cy.get('#messenger-name').should('be.focused');
+    cy.get('#messenger-name').should('have.attr', 'required');
+    cy.get('#email').should('have.attr', 'required');
+    cy.get('#message').should('have.attr', 'required');
+  });
+
+  it('Mostra e oculta uma mensagem de sucesso ao enviar o formulário do messenger com sucesso', () => {
+    cy.clock();
+    cy.getByClassThatStartsWith('Messenger_openCloseButton').click();
+    cy.get('#messenger-name').should('be.visible');
+    
+    cy.get('#messenger-name').type('John Doe');
+    cy.get('#email').type('john.doe@example.com');
+    cy.get('#message').type('Hello, I need support!');
+    cy.getByClassThatStartsWith('Messenger_sendButton').click();
+
+    cy.getByClassThatStartsWith('Messenger_success')
+      .should('be.visible')
+      .and('contain', 'Your message has been sent.');
+    cy.tick(3000);
```

Comment:
```suggestion
    cy.tick(3000);

```

Deixando uma linha pra separar o _assert_.